.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Please add relevant excerpts from the following logs: `vhive`, `firecracker-containerd`, and if used with Kubernetes additionally `containerd` and/or other relevant logs of Kubernetes containers and setup scripts.
+
+**Notes**
+Currently, we support only Ubuntu 18 bare-metal hosts, however we encourage the users to reports Issues that appear in different settings. We will try to help and potentially include these scenarios into our CI if given enough interest from the community.
```

Comment:
Should we add link to the tested cloudlab nodes?

Reply:
no, we support any Ubuntu 18/x86 host.